### ISSUE TITLE
replace build.bat by build.py in osmWebWizard

### ISSUE THREAD

**Issue by behrisch** [eclipse/sumo#7581]
Pros:
- works platform independent and also cross platform (generate on Windows use on Linux etc.)
- easier parallelization of tasks

Cons:
- need to change many tutorials / introductory slides
- starting build.py from the command line or via double click is not as failsafe as starting build.bat (especially on Windows)
- people might be more hesitant to edit a py script than a bat script

Please discuss @namdre @yunpangfloetteroed @schwamborn @RobertHilbrich

**Comment by schwamborn** [comment]
Being a Unix user first and foremost, I am more confused by having a `build.bat` and `run.bat` in my generated scenario folder, where only one of the two has executable access rights and only one of them has the shebang line.

Getting back on-topic, I'm all in for having a python script instead. I think the pros outweigh the cons.

**Comment by namdre** [comment]
I wondering whether the typical windows user has ever seen/used a batch file before. It might be that the hesitation is similar in both cases. 
The fact that people created a scenario with the wizard indicates their ability to execute a python file. Can you think of a reason why one would work and the other would not (in particular for python files without arguments)? Does the installed start-menu-link to the wizard specify the python interpreter?

**Comment by behrisch** [comment]
You are right, if the webWizard works, every other python script should too. The start menu link does not specify a python interpreter.

**Comment by yunpangfloetteroed** [comment]
Hi! actually I agree with the 2nd and the 3rd point in Cons from the point of a window/sumo-user. With batch a user just needs to double click it without typing and/or setting any python-file-linking! It will be also fine to generate both files (*bat  and *sh). But, it seems that mac needs something else than what linux wants (at least on two macs)................

**Comment by behrisch** [comment]
OK, maybe we start then with a bat and a py file and see later whether the bat is still needed?

**Comment by namdre** [comment]
@yunpangfloetteroed If the user started webWizard from a file explorer or via the start menu shortcut it means that double-clicking on python files is working.
